### ISSUE TITLE
Flake8-mypy doesn't report most mypy error

### ISSUE THREAD

**Issue by thibaultdelor** [ambv/flake8-mypy#23]

**Comment by thibaultdelor** [comment]
Running it in verbose mode:
```shell
$ python -m flake8 -v --mypy-config="$(pwd)/mypy.ini" simpleExample.pyflake8.plugins.manager    MainProcess    143 INFO     Loading entry-points for "flake8.extension".
flake8.plugins.manager    MainProcess    146 INFO     Loading entry-points for "flake8.listen".
flake8.plugins.manager    MainProcess    146 INFO     Loading entry-points for "flake8.report".
flake8.plugins.manager    MainProcess    146 INFO     Loading plugin "C90" from entry-point.
flake8.plugins.manager    MainProcess    147 INFO     Loading plugin "F" from entry-point.
flake8.plugins.manager    MainProcess    165 INFO     Loading plugin "pycodestyle.ambiguous_identifier" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.bare_except" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.blank_lines" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.break_after_binary_operator" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.break_before_binary_operator" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.comparison_negative" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.comparison_to_singleton" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.comparison_type" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.compound_statements" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.continued_indentation" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.explicit_line_join" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.extraneous_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.imports_on_separate_lines" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.indentation" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.maximum_line_length" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.missing_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.missing_whitespace_after_import_keyword"from entry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.missing_whitespace_around_operator" fromentry-point.
flake8.plugins.manager    MainProcess    172 INFO     Loading plugin "pycodestyle.module_imports_on_top_of_file" from entry-point.
flake8.plugins.manager    MainProcess    173 INFO     Loading plugin "pycodestyle.python_3000_async_await_keywords" from entry-point.
flake8.plugins.manager    MainProcess    180 INFO     Loading plugin "pycodestyle.python_3000_backticks" from entry-point.
flake8.plugins.manager    MainProcess    180 INFO     Loading plugin "pycodestyle.python_3000_has_key" from entry-point.
flake8.plugins.manager    MainProcess    180 INFO     Loading plugin "pycodestyle.python_3000_invalid_escape_sequence" from entry-point.
flake8.plugins.manager    MainProcess    180 INFO     Loading plugin "pycodestyle.python_3000_not_equal" from entry-point.
flake8.plugins.manager    MainProcess    181 INFO     Loading plugin "pycodestyle.python_3000_raise_comma" from entry-point.
flake8.plugins.manager    MainProcess    181 INFO     Loading plugin "pycodestyle.tabs_obsolete" from entry-point.
flake8.plugins.manager    MainProcess    181 INFO     Loading plugin "pycodestyle.tabs_or_spaces" from entry-point.
flake8.plugins.manager    MainProcess    181 INFO     Loading plugin "pycodestyle.trailing_blank_lines" from entry-point.
flake8.plugins.manager    MainProcess    182 INFO     Loading plugin "pycodestyle.trailing_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    182 INFO     Loading plugin "pycodestyle.whitespace_around_comma" from entry-point.
flake8.plugins.manager    MainProcess    182 INFO     Loading plugin "pycodestyle.whitespace_around_keywords" from entry-point.
flake8.plugins.manager    MainProcess    183 INFO     Loading plugin "pycodestyle.whitespace_around_named_parameter_equals" from entry-point.
flake8.plugins.manager    MainProcess    183 INFO     Loading plugin "pycodestyle.whitespace_around_operator" from entry-point.
flake8.plugins.manager    MainProcess    183 INFO     Loading plugin "pycodestyle.whitespace_before_comment" from entry-point.
flake8.plugins.manager    MainProcess    183 INFO     Loading plugin "pycodestyle.whitespace_before_parameters" from entry-point.
flake8.plugins.manager    MainProcess    183 INFO     Loading plugin "T4" from entry-point.
flake8.plugins.manager    MainProcess    309 INFO     Loading plugin "default" from entry-point.
flake8.plugins.manager    MainProcess    311 INFO     Loading plugin "pylint" from entry-point.
flake8.plugins.manager    MainProcess    311 INFO     Loading plugin "quiet-filename" from entry-point.
flake8.plugins.manager    MainProcess    311 INFO     Loading plugin "quiet-nothing" from entry-point.
flake8.checker            MainProcess    312 INFO     Making checkers
flake8.checker            MainProcess    314 INFO     Checking 1 files
flake8.processor          ForkPoolWorker-1    339 WARNING  Plugin requested optional parameter "visitor" but this is not an available parameter.
flake8.main.application   MainProcess    439 INFO     Finished running
flake8.main.application   MainProcess    439 INFO     Reporting errors
flake8.main.application   MainProcess    439 INFO     Found a total of 0 violations and reported 0
```
